frontend/src/components/TestStatusCard.jsx

```
@@ -134,10 +127,9 @@ const TestStatusCard = ({ variant = 'summary', className = '' }) => {
 
     const fetchLiveAudioVideoStatus = async () => {
         try {
-            const response = await fetch(`${API_BASE_URL}/api/monitoring/live-audio-video-status`);
-            if (response.ok) {
-                const data = await response.json();
-                setLiveAudioVideoStatus(prevStatus => data.data || prevStatus);
+            const response = await spiritualAPI.get('/api/monitoring/live-audio-video-status');
+            if (response && response.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)